tests/nocover/test_strategy_state.py

```diff
@@ -72,13 +72,13 @@ def set_database(self):
         self.teardown()
         self.database = ExampleDatabase()
 
-    @rule(strat=strategies, r=integers(), mshr=integers(0, 100))
-    def find_constant_failure(self, strat, r, mshr):
+    @rule(strat=strategies, r=integers(), max_shrink=integers(0, 100))
```

Comment:
If we're going to change this to a human readable name I feel like we should match the name in settings and have it be max\_shrinks instead of max\_shrink.